og_ui/og_ui.module

```diff
@@ -95,3 +96,26 @@ function og_ui_entity_type_save(EntityInterface $entity) {
     }
   }
 }
+
+/**
+ * Implements hook_entity_type_alter().
+ */
+function og_ui_entity_type_alter(array &$entity_types) {
+//  $group_manager = Og::groupManager();
+  
+  /** @var $entity_types \Drupal\Core\Entity\EntityTypeInterface[] */
+  foreach ($entity_types as $entity_type_id => $entity_type) {
+
+//    if (!$group_manager->getGroupsForEntityType($entity_type_id)) {
+//      continue;
+//    }
+
+    // Adding the group manager causing for a circular dependency because the
+    // group manager the the entity manager as a dependency.
```

Comment:
@pfrenssen @damiankloip Any ideas over here? As far as I can see it we need to remove part of the logic that determine if this is a group or not outside the group manager.